.gitignore

```diff
@@ -2,3 +2,4 @@
 tmp
 .bundle
 vendor
+**/*.gem
```

Comment:
The reason I added it is because a gem was in the repository - probably as the result of using this https://github.com/grosser/dotfiles/blob/master/gitconfig#L5